.circleci/config.yml.erb

```diff
@@ -0,0 +1,84 @@
+# To update the build configuration, edit the "builds" array below and run:
+# erb .circleci/config.yml.erb > .circleci/config.yml
+<%
+  builds = [
+		['2.2.7', 'rails_3'],
+		['2.3.4', 'rails_3'],
+		['2.2.7', 'rails_4'],
+		['2.3.4', 'rails_4'],
+		['2.4.1', 'rails_4'],
+		['2.2.7', 'rails_5'],
+		['2.3.4', 'rails_5'],
+		['2.4.1', 'rails_5'],
+		['2.2.7', 'rails_5_1'],
+		['2.3.4', 'rails_5_1'],
+		['2.4.1', 'rails_5_1'],
+  ]
+%>
+version: 2
+jobs:
+<% builds.each do |ruby,variant| %>
+  build_<%= ruby %>_<%= variant %>:
+    docker:
+      - image: deliveroo/multiruby
+      - image: postgres:9.6-alpine
+      - image: redis:3-alpine
+    steps:
+      - checkout
+
+      - run:
+          name: Setup RAM disk
+          command: |
+            rm -rf ~/project/tmp ;
+            mkdir /dev/shm/tmp ;
+            ln -s /dev/shm/tmp ~/project/tmp
+
+      - run:
+          name: Select build variant (Ruby <%= ruby %>, <%= variant %>)
+          command: |
+            rbenv local <%= ruby %> ;
+            gem install bundler ;
+            bundle config --local gemfile $PWD/gemfiles/<%= variant %>.gemfile
+
+      - restore_cache:
+          keys: 
+            - v2-bundle-<%= ruby %>-<%= variant %>-{{ .Branch }}
+            - v2-bundle-<%= ruby %>-<%= variant %>
+            - v2-bundle-<%= ruby %>
+
+      - run: 
+          name: Install dependencies
+          command: |
+            bundle install --jobs=3 --retry=3 --path=$PWD/vendor/bundle
+
+      - run:
+          name: Run test suite
+          command: |
+            unset RACK_ENV &&
```

Comment:
Missing backslashes?

Reply:
Not necessary thanks to the pipe above.

Reply:
My `irb` interpreter says otherwise:

```
irb(main):001:0> require 'yaml'
=> true
irb(main):002:0> YAML
=> Psych
irb(main):003:0> YAML.load(File.read('hello.yaml'))
=> {"hello"=>"one\ntwo\nthree\n"}
irb(main):004:0> 
marcinw@bruno yaml
$ cat hello.yaml 
hello: |
  one
  two
  three
```

I think what you'd want to use in this case is `>`:

```
irb(main):013:0> require 'yaml'
=> true
irb(main):014:0> YAML.load(File.read('hello.yaml'))
=> {"hello"=>"one two three\n"}
irb(main):015:0> 
marcinw@bruno yaml
$ cat hello.yaml 
hello: >
  one
  two
  three
```

Reply:
I guess it works because Bash treats dangling `&&` as expecting further input:

```
marcinw@bruno yaml
$ unset HELLO &&
> unset BOOM &&
> bundle exec rspec
```